src/components/tutorials/common/steps/advanced-3.js

```diff
@@ -10,10 +10,8 @@ class Advanced3 extends React.PureComponent {
     return (
       <Tabs className={cssClass}>
         <TabList>
-          <Tab>Request</Tab>
           <Tab>Response</Tab>
         </TabList>
-        <TabPanel />
```

Comment:
Were these `<TabPanel />` lines just erroneous close tags...? Oof 🙈

Reply:
Nah, they're the empty TabPanels mapping to the 'Request' tabs

Reply:
Oh whoops for some reason I read these as close tags, not the open+closed combined... I'm an idiot